tmt/steps/provision/bootc.py

```diff
@@ -271,6 +283,8 @@ def _build_bootc_disk(self, containerimage: str, image_builder: str) -> None:
             "build",
             "--type",
             "qcow2",
+            "--rootfs",
+            rootfs,
```

Comment:
So we always set the `rootfs` value. This means that we always override by our default. What if the user wants to keep the default `rootfs` value from the source container? Shall we add a special value `default` for that use case?

Reply:
Probably some kind of `”original”` or `”keep”` default?

Reply:
Yes, something like this. I thought `default` would cover well the `default from the source container`. Maybe it could be confusing. Your `original` sounds good as well. Or we could use `source` if `source container` is the term commonly used.